docs/yuidoc-p5-theme/layouts/main.handlebars

```diff
@@ -197,9 +197,10 @@
 
 
     $(document).ready(function() {
+      var langs = ['es', 'zh-Hans'];
```

Comment:
From what I can find, this is the only spot in the reference pages that handles i18n since the Backbone router doesn't handle the i18n stuff and just the reference entries. The website itself has more potential for consolidation but I want to revisit if I manage to rework the reference creation.

Reply:
aha i see. sounds good